sql/step_36_patientexamination_and_rollback_step23.sql

```diff
@@ -13,7 +13,7 @@ CREATE TABLE PATIENTEXAMINATION (
 	
 	INDEX ( PEX_PAT_ID  ) ,
 	PRIMARY KEY ( PEX_ID )
-);
+) ENGINE=MyISAM;
```

Comment:
I am confused if I look in `load_demo_data.sql` at line 5155 I see:
`) ENGINE=InnoDB AUTO_INCREMENT=50 DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_general_ci;`

I only looked there because I had assumed that the change suggested here should have been made there too.

Reply:
So there must be some place else that the engine type is being set.

Reply:
I tested by building the container with docker compose, which doesn't use `load_demo_data.sql` as it does not load the demo data but takes care of creating the database structure